website/cue/reference/components/transforms/remap.cue

```diff
@@ -8,6 +8,13 @@ components: transforms: "remap": {
 		[Vector Remap Language](\(urls.vrl_reference)) (VRL), an expression-oriented language designed for processing
 		observability data (logs and metrics) in a safe and performant manner.
 
+		This transform also implements an additional `errors` output. When the
+		`drop_on_error` configuration value is set to `true`, events that result in
+		runtime errors will be dropped from the default output stream and sent to
+		the `errors` output instead. For a transform component named `foo`, this
+		error output can be accessed by specifying `foo.errors` as the input to
+		another component.
```

Comment:
This needs to be rewritten to also mention `drop_on_abort`, and rename `errors` output to `dropped`.